.muse

```diff
@@ -1,5 +1,5 @@
 # prefer to build with this environment
-ENVSET p050
+ENVSET p053
```

Comment:
Please put it back to p051.  There is an email thread with the sequence we are using.  Your step is p051.  The step after that will be p053, which adds new the new art, that has yet to be validated.  We want to validate it with your PR merged in.

Reply:
ok, so I dont need to change anything once I've done that?